edn/build.rs

```diff
@@ -0,0 +1,6 @@
+
+extern crate peg;
+
+fn main() {
+    peg::cargo_build("src/edn.rustpeg");
```

Comment:
It's fairly explicitly not rust though. It happens to have a `use` keyword that works like Rust's, along with vaguely Rust like code blocks (except that the braces must match, even in strings), and comments (except that they're 'C' style not Rust style, i.e. multi-line comments don't nest)